aksel.nav.no/website/components/website-modules/search/utils/format-result.ts

```diff
@@ -1,8 +1,8 @@
-import Fuse from "fuse.js";
+import type { FuseResult, FuseResultMatch } from "fuse.js";
 import { omit } from "@navikt/ds-react";
 import { FuseItemT, SearchHitT } from "@/types";
 
-export function formatResults(res: Fuse.FuseResult<FuseItemT>[]): SearchHitT[] {
+export function formatResults(res: FuseResult<FuseItemT>[]): SearchHitT[] {
```

Comment:
Gjorde disse endringene for å fikse følgende feilmelding lokalt:
![image](https://github.com/navikt/aksel/assets/83693529/a15cdb0a-2c53-4f71-897c-2be45387c3ad)